bridge/README.md

```diff
@@ -0,0 +1,39 @@
+# Bridge SDK
+
+## Contents
+
+- [Compatibility](#compatibility)
+- [Versioning](#versioning)
+
+## Compatibility
+
+- **Minimum SDK**: 28
+- **Target SDK**: 34
+- **Device Types Supported**: Phone and Tablet
+- **Orientations Supported**: Portrait and Landscape
+
+## Versioning
+This repository conforms to the following versioning convention:
+
+**v[RELEASE].[INTERNAL].[PATCH]**
```

Comment:
This make this

```
**v[MAJOR].[MINOR].[PATCH]**
```

Reply:
Yep, update this and corresponding references as well.